.github/workflows/update-specs-and-client-libraries.yaml

```diff
@@ -119,6 +125,12 @@ jobs:
               npm install
             version: ${{ needs.generate_specs_and_libraries.outputs.typescript_axios_version }}
             update: ${{ inputs.update-onfido-node }}
+          - generator: php
+            git_repo_id: onfido-php
+            preCommit: |
+              composer install
+            version: ${{ needs.generate_specs_and_libraries.outputs.php_version }}
+            update: ${{ inputs.update-onfido-php }}
```

Comment:
I think we already discussed this, but will there be cases where a single lib is updated? 🤔

Reply:
Yeah, two cases on top of my mind:
- if changing one or more templates (which are language specific)
- if given change to openapi spec is causing mayor change in one client lib and minor to another one (rare case)

Reply:
Another one:
- Update the openapigenerator version and causes major/minor changes in different libs